bot/seasons/halloween/__init__.py

```diff
@@ -3,14 +3,11 @@
 
 class Halloween(SeasonBase):
     name = "halloween"
-    start_date = "01/10"
-    end_date = "31/10"
     bot_name = "Spookybot"
+    greeting = "Happy Halloween!"
 
-    def __init__(self, bot):
-        self.bot = bot
+    start_date = "01/10"
+    end_date = "31/10"
 
-    @property
-    def bot_avatar(self):
-        with open(self.avatar_path("spooky.png"), "rb") as avatar:
-            return bytearray(avatar.read())
+    colour = 0xe67e22
```

Comment:
I feel like this should probably be a constant in the Colours dataclass?

Reply:
Yep, that makes sense. I've added it in.